pooch/tests/test_downloaders.py

```diff
@@ -48,6 +48,17 @@
 ZENODOURL = pooch_test_zenodo_url()
 
 
+@pytest.mark.parametrize("url", [
```

Comment:
This test includes HTTPS, DOI, and SFTP downloaders. 
FTP is missing, because I don't see a nice way to integrate the extra setup it needs into this pytest parameterize list. You *could* manually close the FTP server if needed at the end of the pytest function, but then things get untidy if it encounters an error before that point. You'd do this if you wanted a standalone test, but I'm not sure it's worth the extra maintenance effort to include, tbh
```python
@pytest.mark.network
def test_progressbar_kwarg_passed(ftpserver):
    "Test ftp downloader"
    with data_over_ftp(ftpserver, "tiny-data.txt") as url:
    d = choose_downloader(url, progressbar=True)
    assert d.progressbar is True
```

Reply:
Anyway, if you think this is insufficient, please leave a comment with a suggestion.

Reply:
@GenevieveBuckley that will be good enough, you're right that it wouldn't be worth the extra effort.